games-board/stockfish/stockfish-sf_8.recipe

```diff
@@ -0,0 +1,67 @@
+SUMMARY="A chess engine claimed to be strongest in the world"
+DESCRIPTION="Stockfish is a free UCI chess engine derived from Glaurung 2.1. It \
+is not a complete chess program and requires some UCI-compatible GUI (e.g. \
+XBoard with PolyGlot, eboard, Arena, Sigma Chess, Shredder, ChessPartner or \
+Fritz) in order to be used comfortably. Read the documentation for your GUI of \
+choice for information about how to use Stockfish with it."
+HOMEPAGE="https://www.stockfishchess.org/"
+COPYRIGHT="2004-2008 Tord Romstad (Glaurung author)
+	2008-2017 Marco Costalba, Joona Kiiski, Gary Linscott, Tord Romstad"
```

Comment:
sf_8 was released on November 1st, 2016, so you should replace 2017 by 2016.

BTW, we should probably rename the recipe to stockfish-8.recipe and use "sf_$portVersion" where we are using "$portVersion".
You would then change SOURCE_DIR to
```
SOURCE_DIR="Stockfish-sf_$portVersion"
```

To rename:
```
cd haikuports/games-board/stockfish
git mv stockfish-sf_8.recipe stockfish-8.recipe
git commit --amend --no-edit stockfish-sf_8.recipe stockfish-8.recipe
```